src/iccpd/src/iccp_ifm.c

```diff
@@ -502,7 +502,7 @@ int do_one_neigh_request(struct nlmsghdr *n)
 {
     struct ndmsg *ndm = NLMSG_DATA(n);
     int len = n->nlmsg_len;
-    struct rtattr * tb[NDA_MAX + 1];
+    struct rtattr *tb[NDA_MAX + 1] = {NULL};
```

Comment:
It looks like line 969 is also not zeroing out the struct. Suggest making the same change there, as well.

Also, suggest changing from `= {NULL};` to `= { 0 };`. Theoretically the behavior is the same, but the latter is more commonly used.